src/handlers/comment/handlers/comment-incentives.ts

```diff
@@ -0,0 +1,23 @@
+import { getBotContext, getLogger } from "../../../bindings";
+import { Payload } from "../../../types";
+
+export const commentIncentive = async (body: string) => {
+  const context = getBotContext();
+  const logger = getLogger();
+  const payload = context.payload as Payload;
+  const sender = payload.sender.login;
+
+  logger.info(`Received '/comment-incentive' command from user: ${sender}`);
+
+  if (!payload.issue) {
+    logger.info(`Skipping '/comment-incentive' because of no issue instance`);
+    return `Skipping '/comment-incentive' because of no issue instance`;
+  }
+  const parts = body.split(" ");
+  parts.shift();
+  if (parts.pop() !== "true" && parts.pop() !== "false") {
```

Comment:
you can create a helper function that parses the command so you can use it here and in `getIncentivizedUsers` to avoid repeating code